wasm/lib/src/vm_class.rs

```diff
@@ -44,7 +57,7 @@ impl StoredVirtualMachine {
             setup_browser_module(&vm);
         }
 
-        import::init_importlib(&vm, false).unwrap();
+        vm.initialize(false);
```

Comment:
This call can be removed. Set the setting `initialize_as_external` to `Some(false)`.

Reply:
The VM initialize needs to after then`js_module::setup_js_module` and `setup_browser_module`.  So this public function calls here and not in the timing of VM creating.

In wasm, the process is creating VM, injecting, and then initialize as external.

The `initialize(false)` is misleading I will change this to `initalize_as_external` latter.

Reply:
The `initialized_as_external` may be ambiguous, the `initialize_with_external_importer` is adopted for clearness.
Thanks.